### ISSUE TITLE
Issue with pagination link generation

### ISSUE THREAD

**Issue by alex-LE** [stevebauman/eloquent-table#7]
Hey,
once i add the "showPages" function call, i get the following error:
```
Call to undefined method Stevebauman\EloquentTable\TableCollection::appends()
```
Also with the manual call for the links i get an error:
```
call_user_func_array() expects parameter 1 to be a valid callback, class 'Stevebauman\EloquentTable\TableCollection' does not have a method 'links'
```
I'm using Laravel 5 btw. With the showPages links everything works fine.

Alex

**Comment by stevebauman** [comment]
Hi, are you paginating your model results? Such as: `Users::paginate(25)`? The results need to be paginated for the methods `appends()` and `links()` to exist on the collection object since they return different collection instances.

Could you also show the controller / view code?

**Comment by alex-LE** [comment]
sure the controller function looked like this:
```
public function index()
{
	$giftcards = Giftcard::sort(Input::get('field'), Input::get('sort'))->paginate(3);
	return view('admin.giftcard_list', compact('giftcards'));
}
```

(the paginate with 3 was just for testing)

**Comment by stevebauman** [comment]
Ok, can you show your view code where you're rendering the table please?

**Comment by stevebauman** [comment]
Assuming this issue is fixed due. Please leave comment if you're still experiencing this issue.

**Comment by stevebauman** [comment]
You're right, through testing I've received this as well. Will be fixed shortly.

**Comment by stevebauman** [comment]
Work around for the time being, don't use the `showPages()` method, and render your pages yourself on Laravel 5 using `$items->render()`. Pagination was completely revamped in 5, and most methods were renamed. Still trying to work this out.
